goodreads/views.py

```diff
@@ -194,15 +194,10 @@ def populateAuthors(df):
     return authors
 
 def upload(request):
-    template = "goodreads/csv_upload.html"
+    #template = "goodreads/csv_upload.html"
```

Comment:
I wanted to decouple the backend uploading from the front end display page - not sure this is the right approach

Reply:
I agree that you should not need a template for an endpoint that is only called via AJAX. I'm not sure what the recommended way to do that in Django would be.